internal/pkg/service/buffer/sink/tablesink/storage/repository/volume_scheme.go

```diff
@@ -0,0 +1,32 @@
+package repository
+
+import (
+	. "github.com/keboola/keboola-as-code/internal/pkg/service/buffer/sink/tablesink/storage"
+	. "github.com/keboola/keboola-as-code/internal/pkg/service/common/etcdop"
```

Comment:
See: https://stackoverflow.com/questions/6478962/what-does-the-dot-or-period-in-a-go-import-statement-do

It's a bit of a controversial thing. 
I use it only in places where there are only a few imports and it is useful to have it written shorter - more clearly.

I think there are 2 places where it is used:
- Here in the scheme
- And in API definitions, see `/api/...` directory.

What these 2 sites have in common is that they are some `interface definition` or `domain specific language` files.

--------------------------

Practically, there is no `etcd.PrefixT` in the code, but only `PrefixT`.

Reply:
It would like to write down all the uses and discuss it.

Instead of a black/white vision, I would prefer that we agree when it is appropriate and when it is not, since opinions are in generally different.

I created a task in backlog:

https://keboola.atlassian.net/browse/PSGO-347